MonoGame.Framework/Platform/Native/TitleContainer.Interop.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.IO;
+using MonoGame.Interop;
+using System.Runtime.InteropServices;
+
+internal unsafe class MGReadOnlyStream : Stream
```

Comment:
Have we done this sort of prefixing of "MG" in C# classes before?

Maybe this should be `MG_AssetStream` or `AssetStream` ?

Minor thing.